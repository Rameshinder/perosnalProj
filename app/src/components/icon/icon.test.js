import React from 'react'
import { shallow } from 'enzyme'

import Icon from './index'

describe('Icon', () => {
  const app = shallow(<Icon iconName='Check' />)

  test('renders properly', () => {
    expect(app).toMatchSnapshot()
  })

  it('Icon should render without crashing', () => {
    shallow(<Icon iconName='Check' />)
  })
})
