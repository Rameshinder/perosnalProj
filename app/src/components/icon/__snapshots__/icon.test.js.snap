// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Icon
    iconName="Check"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <path
        d="M13.6131255,27.0121555 C13.1131255,27.0121555 12.6131255,26.8121555 12.2131255,26.5121555 L4.61312546,19.3121555 C3.81312546,18.5121555 3.81312546,17.3121555 4.51312546,16.5121555 C5.31312546,15.7121555 6.51312546,15.7121555 7.31312546,16.4121555 L13.2131255,22.0121555 L24.4131255,6.41215551 C25.0131255,5.51215551 26.2131255,5.21215551 27.2131255,5.81215551 C28.1131255,6.41215551 28.4131255,7.61215551 27.8131255,8.61215551 L15.3131255,26.2121555 C15.0131255,26.7121555 14.4131255,27.0121555 13.9131255,27.1121555 C13.8131255,27.0121555 13.7131255,27.0121555 13.6131255,27.0121555 Z"
        fill="#0072AC"
      />,
      "height": "1em",
      "viewBox": "0 0 32 32",
      "width": "1em",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "d": "M13.6131255,27.0121555 C13.1131255,27.0121555 12.6131255,26.8121555 12.2131255,26.5121555 L4.61312546,19.3121555 C3.81312546,18.5121555 3.81312546,17.3121555 4.51312546,16.5121555 C5.31312546,15.7121555 6.51312546,15.7121555 7.31312546,16.4121555 L13.2131255,22.0121555 L24.4131255,6.41215551 C25.0131255,5.51215551 26.2131255,5.21215551 27.2131255,5.81215551 C28.1131255,6.41215551 28.4131255,7.61215551 27.8131255,8.61215551 L15.3131255,26.2121555 C15.0131255,26.7121555 14.4131255,27.0121555 13.9131255,27.1121555 C13.8131255,27.0121555 13.7131255,27.0121555 13.6131255,27.0121555 Z",
        "fill": "#0072AC",
      },
      "ref": null,
      "rendered": null,
      "type": "path",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <path
          d="M13.6131255,27.0121555 C13.1131255,27.0121555 12.6131255,26.8121555 12.2131255,26.5121555 L4.61312546,19.3121555 C3.81312546,18.5121555 3.81312546,17.3121555 4.51312546,16.5121555 C5.31312546,15.7121555 6.51312546,15.7121555 7.31312546,16.4121555 L13.2131255,22.0121555 L24.4131255,6.41215551 C25.0131255,5.51215551 26.2131255,5.21215551 27.2131255,5.81215551 C28.1131255,6.41215551 28.4131255,7.61215551 27.8131255,8.61215551 L15.3131255,26.2121555 C15.0131255,26.7121555 14.4131255,27.0121555 13.9131255,27.1121555 C13.8131255,27.0121555 13.7131255,27.0121555 13.6131255,27.0121555 Z"
          fill="#0072AC"
        />,
        "height": "1em",
        "viewBox": "0 0 32 32",
        "width": "1em",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M13.6131255,27.0121555 C13.1131255,27.0121555 12.6131255,26.8121555 12.2131255,26.5121555 L4.61312546,19.3121555 C3.81312546,18.5121555 3.81312546,17.3121555 4.51312546,16.5121555 C5.31312546,15.7121555 6.51312546,15.7121555 7.31312546,16.4121555 L13.2131255,22.0121555 L24.4131255,6.41215551 C25.0131255,5.51215551 26.2131255,5.21215551 27.2131255,5.81215551 C28.1131255,6.41215551 28.4131255,7.61215551 27.8131255,8.61215551 L15.3131255,26.2121555 C15.0131255,26.7121555 14.4131255,27.0121555 13.9131255,27.1121555 C13.8131255,27.0121555 13.7131255,27.0121555 13.6131255,27.0121555 Z",
          "fill": "#0072AC",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
